pkg/controller/controller_test.go

```diff
@@ -97,6 +102,56 @@ func TestController(t *testing.T) {
 				},
 			},
 			expectedActions: sets.NewString("DELETE=/api/v1/namespaces/n1/configmaps/map1"),
+			namespace:       meta_v1.NamespaceNone,
```

Comment:
Could you change it to `meta_v1.NamespaceAll`? Both are empty strings but `meta_v1.NamespaceAll` means "(watch) all namespaces". `meta_v1.NamespaceNone` should be used as namespace for cluster-wide objects. I.e. meaning of the constant is different.